### PR TITLE
Add security page

### PR DESCRIPTION
**Pull request by 4p00rv**

### CODE REVIEW

File: src/templates/user/security.html.tt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                        <div class="gr-12 rbox-content" id="client_message_content">
+                            <p class="center notice-msg"></p>
+                        </div>
+                    </div>
+                </div>
+
+                <div id="fx-security-content" style="display: none">
+                    <div class="row">
+                        <div class="col-md-3">
+                            <a href="[% request.url_for('user/security/change_password') %]">
```
Comment (mohammad-deriv) [inline src/templates/user/security.html.tt]: It's `user/change-password`

File: src/templates/user/security.html.tt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                                <h3>[% l('Account password') %]</h3>
+                            </a>
+                            <div class="separator primary-color"></div>
+                            <p>[% l('Change your ChampionFX account login password.') %]</p>
+                        </div>
+                        <div class="col-md-3 offset-xs-1 fx-real" style="display: none">
+                            <a href="[% request.url_for('user/security/cashier_password') %]">
```
Comment (mohammad-deriv) [inline src/templates/user/security.html.tt]: `cashier/cashier-password`

File: src/templates/user/security.html.tt [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                                <h3>[% l('Cashier password') %]</h3>
+                            </a>
+                            <div class="separator primary-color"></div>
+                            <p>[% l('Create or change your cashier password to restrict deposits and withdrawals.') %]</p>
+                        </div>
+                        <div class="col-md-3 offset-xs-1">
```
Comment (mohammad-deriv) [inline src/templates/user/security.html.tt]: Please hide this section, as this is not ready yet.
Thanks.

**Comment by 4p00rv** [comment]
@binary-hamedanchi I have made the changes that you pointed out.

**Review by mohammad-deriv — APPROVED** [review]

**Pull request merged**